strictdoc/backend/excel/import_/excel_sheet_proxy.py

```diff
@@ -0,0 +1,70 @@
+from typing import List
+
+import openpyxl
+import xlrd
+
+from strictdoc.helpers.auto_described import auto_described
+
+
+@auto_described
+class ExcelSheetProxy:
+    """This proxy class allows to open either xls(xlrd) or xlsx(openpyxl) with the same interface"""
+
+    def __init__(self, file: str):
+        self.file = file
+        if file.endswith(".xlsx"):
+            self.workbook = openpyxl.load_workbook(file)
+            self.sheet = self.workbook.active
+            self.lib = "openpyxl"
+        elif file.endswith(".xls"):
+            self.workbook = xlrd.open_workbook(file, on_demand=True)
+            self.sheet = self.workbook.sheet_by_index(0)
+            self.lib = "xlrd"
+        else:
+            raise ValueError("Unsupported file format")
+
+    def name(self) -> str:
```

Comment:
Why not make name, ncols, nrows `@property`? Then calling code (e.g. in `excel_to_sdoc_converter.py`) doesn't have to change to add `()`.